docs/tutorial/part_1.rst

```diff
@@ -188,25 +188,25 @@ Your app directory should now look like::
 In the ``chat/urls.py`` file include the following code::
 
     # chat/urls.py
-    from django.conf.urls import url
+    from django.urls import path
     
     from . import views
     
     urlpatterns = [
-        url(r'^$', views.index, name='index'),
+        path('', views.index, name='index'),
     ]
 
 The next step is to point the root URLconf at the **chat.urls** module.
 In ``mysite/urls.py``, add an import for **django.conf.urls.include** and
 insert an **include()** in the **urlpatterns** list, so you have::
 
     # mysite/urls.py
-    from django.conf.urls import include, url
     from django.contrib import admin
-    
+    from django.urls import path,include
+
     urlpatterns = [
-        url(r'^chat/', include('chat.urls')),
-        url(r'^admin/', admin.site.urls),
+        path('chat', include('chat.urls')),
```

Comment:
Is there a reason you removed the slash (`chat` instead of `chat/`?). If not, please add it back.